hoodie-client/src/main/java/com/uber/hoodie/io/HoodieIOHandle.java

```diff
@@ -65,10 +69,12 @@ public Path makeNewPath(String partitionPath, int taskPartitionId, String fileNa
         FSUtils.makeDataFileName(commitTime, taskPartitionId, fileName));
   }
 
-  public Path makeTempPath(String partitionPath, int taskPartitionId, String fileName, int stageId, long taskAttemptId) {
+  public Path makeTempPath(String partitionPath, int taskPartitionId, String fileName, int stageId,
```

Comment:
May I suggest having your autosave option to only format changed lines? (if thats doable)